exchanges/gateio/gateio_test.go

```diff
@@ -1809,8 +1809,10 @@ func TestUpdateTickers(t *testing.T) {
 func TestUpdateOrderbook(t *testing.T) {
 	t.Parallel()
 	for _, a := range g.GetAssetTypes(false) {
-		_, err := g.UpdateOrderbook(t.Context(), getPair(t, a), a)
-		assert.NoErrorf(t, err, "UpdateOrderbook should not error for %s", a)
+		o, err := g.UpdateOrderbook(t.Context(), getPair(t, a), a)
+		require.NoErrorf(t, err, "UpdateOrderbook must not error for %s", a)
+		assert.NotEmptyf(t, o.Bids, "UpdateOrderbook should return some bids for %s", a)
+		assert.NotEmptyf(t, o.Asks, "UpdateOrderbook should return some asks for %s", a)
```

Comment:
🦍 👑   Consider `t.Run` instead of `assquire.*f`

Reply:
https://github.com/thrasher-corp/gocryptotrader/pull/1917/commits/f5a6df3e746b59be6ca20f535f736ac1f5b59304